samples/RazorSlices.Samples.WebApp/Program.cs

```diff
@@ -85,6 +86,37 @@
         : TypedResults.NotFound();
 });
 
+
+var htmxTodos = app.MapGroup("/htmx-todo");
+var renderTodoTable = () => Results.Extensions.RazorSlice<Slices.HtmxTodos._Todos, List<Models.Todo>>(Models.Todos.AllTodos);
```

Comment:
Why are you declaring this delegate variable here this way? Can you just follow the regular pattern of invoking the results helper extension in the handler delegate body?